src/Raven.Server/Dashboard/Cluster/Notifications/NotificationsSummaryNotificationSender.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Threading;
+using Raven.Server.Json;
+using Raven.Server.NotificationCenter;
+using Raven.Server.NotificationCenter.Notifications;
+using Sparrow.Json;
+
+namespace Raven.Server.Dashboard.Cluster.Notifications;
+
+public class NotificationsSummaryNotificationSender : AbstractClusterDashboardNotificationSender
+{
+    private readonly DatabasesInfoRetriever _databasesInfoRetriever;
+    private NotificationsSummaryRequestConfig _notificationsSummaryRequestConfig;
+    
+    public NotificationsSummaryNotificationSender(int widgetId, DatabasesInfoRetriever databasesInfoRetriever, ConnectedWatcher watcher, BlittableJsonReaderObject configuration, CancellationToken shutdown) : base(widgetId, watcher, shutdown)
+    {
+        _databasesInfoRetriever = databasesInfoRetriever;
+        _notificationsSummaryRequestConfig = JsonDeserializationServer.NotificationsSummaryRequestConfig(configuration);
+    }
+
+    protected override TimeSpan NotificationInterval { get; } = TimeSpan.FromSeconds(5);
+    
+    protected override AbstractClusterDashboardNotification CreateNotification()
+    {
```

Comment:
what is the performance impact on doing that every 5 seconds? consider low tier cloud instances  & multi tenant usage